frontend/src/common/analytics/events.ts

```diff
@@ -72,12 +72,21 @@ export enum EVENTS {
   WMG_FMG_GENE_INFO = "WMG_FMG_GENE_INFO",
   WMG_SOURCE_DATA_CLICKED = "WMG_SOURCE_DATA_CLICKED",
   WMG_SOURCE_DOCUMENTATION_CLICKED = "WMG_SOURCE_DOCUMENTATION_CLICKED",
-  CELL_CARDS_CLICK_NAV = "CELL_CARDS_CLICK_NAV",
+  CELL_GUIDE_CLICK_NAV = "CELL_GUIDE_CLICK_NAV",
   NEWSLETTER_EMAIL_SUBMITTED = "NEWSLETTER_EMAIL_SUBMITTED",
   NEWSLETTER_SIGNUP_SUCCESS = "NEWSLETTER_SIGNUP_SUCCESS",
   NEWSLETTER_SIGNUP_FAILURE = "NEWSLETTER_SIGNUP_FAILURE",
   NEWSLETTER_OPEN_MODAL_CLICKED = "NEWSLETTER_OPEN_MODAL_CLICKED",
   NEWSLETTER_DIRECT_LINK_NAVIGATED = "NEWSLETTER_DIRECT_LINK_NAVIGATED",
   NEWSLETTER_SUBSCRIBE_BUTTON_AVAILABLE = "NEWSLETTER_SUBSCRIBE_BUTTON_AVAILABLE",
   WMG_CLEAR_GENES_CLICKED = "WMG_CLEAR_GENES_CLICKED",
+  CG_SEARCH_CT_TISSUE = "CG_SEARCH_CT_TISSUE",
```

Comment:
I assume we still need to add these events in Plausible 😄 ? (I didn't see the new event names in Plausible, yet) Thank you!

Reply:
Ah I see the checklist!

https://github.com/chanzuckerberg/single-cell-data-portal/issues/5049#issue-1759664808